### PR TITLE
Updating the Broken link to "Node.js Contribution Guidelines" in Contributing page

### LINKED ISSUE #5296: Broken link to "Node.js Contribution Guidelines" in Contributing page
### URL:

https://nodejs.org/en/get-involved/contribute

### Browser Name:

Chrome

### Browser Version:

112.0.5615.137

### Operating System:

MacOS Monterey 12.2

### How to reproduce the issue:

1. if I go to the https://nodejs.org/en/get-involved/contribute page and then click on the "Node.js Contribution Guidelines" link
2. I expect to see the Node.js Contribution Guidelines page
3. I see the GitHub 404 page

The actual link is https://github.com/nodejs/node/blob/main/CONTRIBUTING#pull-requests,
it should be https://github.com/nodejs/node/blob/main/CONTRIBUTING.md#pull-requests

### PR DESCRIPTION
**Pull request by vasanth9**
Fixes #5296

### CODE REVIEW

**Review by mikeesto — APPROVED** [review]
Thanks for your first contribution

File: pages/en/get-involved/contribute.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The Node.js project is currently managed across a number of separate GitHub repo
 
 ## Code contributions
 
-If you'd like to fix bugs or add a new feature to Node.js, please make sure you consult the [Node.js Contribution Guidelines](https://github.com/nodejs/node/blob/main/CONTRIBUTING.md#pull-requests). The review process by existing collaborators for all contributions to the project is explained there as well.
+If you'd like to fix bugs or add a new feature to Node.js, please make sure you consult the [Node.js Contribution Guidelines](https://github.com/nodejs/node/blob/main/CONTRIBUTING.md/#pull-requests). The review process by existing collaborators for all contributions to the project is explained there as well.
```
Comment (shanpriyan) [inline pages/en/get-involved/contribute.md:31]: I think the link (https://github.com/nodejs/node/blob/main/CONTRIBUTING.md#pull-requests) is not broken, I'm able to navigate to it

Comment (ovflowd) [reply pages/en/get-involved/contribute.md:31]: I can reproduce it on the nodejs.org website

image

It seems to be removing the `.md`

**Comment by ovflowd** [comment]
I'm curious why the Vercel bot didn't made a preview branch here? 🤔

**Review by ovflowd — APPROVED** [review]
I ack that this PR fixes the issue. Thanks for the first contribution!

**Comment by bicccio** [comment]
It's not clear why it's required to add a slash after `.md`  to have a working link on the web page.

**Comment by ovflowd** [comment]
Probably some shenanigans from MDX who knows

**Pull request merged**